lib/OctokitAction.ts

```diff
@@ -30,6 +34,22 @@ export abstract class OctokitAction extends Action {
     return this.graphqlWithAuth(query);
   }
 
+  protected async moveIssue(issueId: string, transitionName: string): Promise<void> {
+    try {
+      console.log(`${issueId}: Getting transition id for '${transitionName}'`);
+      let response = await this.jiraTransitionRequest("GET", issueId);
+      const transition = response.transitions.find((t: any) => t.name === transitionName);
+      if (transition == null) {
+        throw new Error(`Could not find the transition '${transitionName}'`);
+      }
+      console.log(`${issueId}: Executing '${transitionName}' (${transition.id}) transition`);
+      await this.jiraTransitionRequest("POST", issueId, { transition: { id: transition.id } });
+      console.log(`${issueId}: Transition '${transitionName}' successfully excecuted.`);
+    } catch (error) {
+      console.warn(`${issueId}: Failed to move issue with the '${transitionName}' transition: ${error}`);
+    }
```

Comment:
The catch shouldn't be needed on happy and unhappy path, unless the token is wrong